libs/island-ui/core/src/lib/Toast/Toast.css.ts

```diff
@@ -10,7 +10,7 @@ export const root = style(
 export const useMotion = style({}, 'useMotion')
 
 globalStyle(`${root} .Toastify__toast-container`, {
-  zIndex: 9999,
+  zIndex: 99999,
```

Comment:
Would be nice to transition to the new zIndex constants in theme 🙈

Reply:
The problem i faced here was that when opening a toast inside a modal, the toast always renders behind the modal because Reakit is forcing a z-index of some ridiculous (9999+) value on all it's modals

I can see that we have zIndex constants `belowModal: 19` `modal: 20` `aboveModal: 21`
These are only used when rendering a Reakit `Menu`, but I guess `Modal` is different

I guess we could add a new constant `aboveEverything: 99999` ?